models/articles.js

```diff
@@ -0,0 +1,28 @@
+const db = require("../db/connection");
+
+exports.selectAllArticles = () => {
+  const selectQuery = `
+    SELECT articles.author,articles.title,articles.article_id ,articles.topic,articles.created_at,articles.votes, COUNT(articles.article_id) AS comment_count FROM articles JOIN  users ON articles.author=users.username
+    LEFT JOIN comments ON articles.article_id=comments.article_id

```

Comment:
Overlooked on a previous merge request, but we want to be counting the number of comments associated with each review - right now when a review has 0 comments we are getting back a count of one, probably because we are counting instances of article ids instead. Counting the comments.comment_id (so each instance of a comment) here should fix this so just a minor change to make!